mappings/net/minecraft/entity/Tameable.mapping

```diff
@@ -1,3 +1,4 @@
 CLASS net/minecraft/unmapped/C_lkirduch net/minecraft/entity/Tameable
+	METHOD m_bllryzha getEntityView ()Lnet/minecraft/unmapped/C_nyckmiua;
```

Comment:
get entity view from who? i unfortunately couldn't confirm it myself when i strumbled on it, but i suspect it is the owner's entity view

Reply:
good catch, I'll double check and specify

Reply:
purely based on the implementation of `getOwner`, I think the entity view belongs to the original entity
despite not being able to locate implementations

Reply:
ah i see